Composer/packages/ui-plugins/virtual-assistant-creation/src/shared/util.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import axios from 'axios';
+import { ShellApi } from '@bfc/shared';
+
+import { AvailablePersonalities } from '../models/creationOptions';
+
+// Utility constants
+const GitHubRepoApiRootUrl =
+  'https://api.github.com/repos/microsoft/BotBuilder-PersonalityChat/contents/CSharp/Datasets/qnaFormat/';
```

Comment:
@cwhitten - Should we use aka.ms links to avoid hard-coding any specific URLs into Composer? In the case it changes, we could then update the URL on aka.ms and not have to redeploy and upgrade everyone.  What do you think?

Reply:
Yeah that would make sense. If its any consolation this hosting method/url is not the long term hosting plan for the personality chat assets. As we move to the asset library model the hosting and storage of these personality .qna files will change. 
This initial implementation works and assumes a hosted github scenario but they will change on the next iteration so I dont think it matters that it is not behind an aka.ms link.